assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/fields/usePrice.tsx

```diff
@@ -0,0 +1,29 @@
+import { useCallback, useMemo } from 'react';
+
+import { useDataState } from '../data';
+import { BaseFieldProps, PriceFieldProps, UsePrice } from './types';
+
+type BFP = BaseFieldProps;
+
+const usePrice = ({ field, price }: PriceFieldProps): UsePrice => {
```

Comment:
Since this hook only deals with price amount, it's better to name it something like `usePriceAmount`

Reply:
great suggestion, updated